FlowCrypt/Resources/pass_phrase_hint.html

```diff
@@ -0,0 +1,50 @@
+<!--
+  ~ © 2016-present FlowCrypt a.s. Limitations apply. Contact human@flowcrypt.com
+  ~ Contributors: DenBond7
+  -->
+<style>
+    body {
+        font-size: 60px;
```

Comment:
let's use `font-size: 48px`, as 60px looks too big in comparison with our common UI fonts

Reply:
ok